packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -766,6 +770,11 @@ export const Dropdown: ComponentWithAs<'div', DropdownProps> &
         render: renderSelectedItem,
       }),
     );
+    return (
+      <div role="listbox" aria-label={a11ySelectedItemsListboxMessage && a11ySelectedItemsListboxMessage}>
```

Comment:
I thing we should use `Box` to create this container rather the element directly and possible pass the `aria-label` to behavior.

Reply:
```suggestion
      <div role="listbox" aria-label={a11ySelectedItemsListboxMessage}>
```

@assuncaocharles there is no sense to have `Box` there as it's not a real slot.

---

@kolaps33 As I see other props related to A11y are functions, should it be `getA11ySelectedItemsListboxMessage()`?

Reply:
@layershifter I was looking to it and I think it will be enough if it is simple "string" prop, because others a11y need be function but this one is only simple one which will add "string"  as "aria-label".
And I renamed it into:
a11ySelectedItemsMessage
as I think `listbox` was not useful in the name.